### PR TITLE
Downgrade python to 3.9 to ensure capture Scrypt

### PR DESCRIPTION
**Pull request by expatjedi**
The wheels available for scrypt are only covered up to python3.9, so ensure our environment understands this and also capture scrypt during binary generation.

### CODE REVIEW

File: contrib/build-wine/build-electrum-git.sh
Comment (BlackcoinDev) [inline contrib/build-wine/build-electrum-git.sh:1]: why remove this?  try this

--no-binary :all: --only-binary cffi,cryptography,PyQt5,PyQt5-Qt5,PyQt5-sip,script

File: contrib/deterministic-build/requirements-binaries-mac.txt
Comment (BlackcoinDev) [inline contrib/deterministic-build/requirements-binaries-mac.txt:1]: can you upgrade to 0.8.20? here are the hashes, I tested this and it works

https://pastebin.com/k4BRGpwj

File: contrib/deterministic-build/requirements-binaries.txt
Comment (BlackcoinDev) [inline contrib/deterministic-build/requirements-binaries.txt:1]: same here 

https://pastebin.com/k4BRGpwj

File: contrib/deterministic-build/requirements-binaries-mac.txt
Comment (expatjedi) [reply contrib/deterministic-build/requirements-binaries-mac.txt:1]: Done

File: contrib/deterministic-build/requirements-binaries.txt
Comment (expatjedi) [reply contrib/deterministic-build/requirements-binaries.txt:1]: done

File: contrib/build-wine/build-electrum-git.sh
Comment (expatjedi) [reply contrib/build-wine/build-electrum-git.sh:1]: --no-binary:all is deprecated and will be remove soon.

**Comment by lateminer** [comment]
`develop` branch got updated, could you please rebase your commits?

**Pull request closed without merging**